ui/src/main/js/common/util/enums/User.js

```diff
@@ -0,0 +1,10 @@
+const UserEnum = {
```

Comment:
I've updated this to reflect how we have been doing it with our newer 'concrete' models.  Initially this was put in place to build with, but your comment made me realize how it could be taken as an enumeration of default users.  I've also removed it from this directory and it is now the UserModel within the usermgmt.user directory.